app/models/language.rb

```diff
@@ -1,8 +1,9 @@
 class Language < ApplicationRecord
   include WorksOwner
-  validates_presence_of :short
-  validates :short, uniqueness: true
-  validates_presence_of :name
+  validates :short, presence: true
+  validates :short, uniqueness: true, length: { maximum: 5 }
+  validates :name, uniqueness: true
+  validates :name, presence: true
```

Comment:
I think these can be combined into 1 `validates` per column
```suggestion
  validates :short, presence: true, uniqueness: true, length: { maximum: 5 }
  validates :name, uniqueness: true, presence: true
```